requirements/adjust-versions.py

```diff
@@ -5,7 +5,8 @@
 from typing import Dict, Optional
 
 VERSIONS = [
-    dict(torch="1.9.0", torchvision="0.10.0", torchtext=""),  # nightly
+    dict(torch="1.10.0", torchvision="0.11.0", torchtext=""),  # nightly
+    dict(torch="1.9.0", torchvision="0.10.0", torchtext=""),
```

Comment:
why don't you set the torchtext version?

Reply:
it is not available I guess https://github.com/pytorch/text#installation